config/development.yaml

```diff
@@ -73,3 +73,12 @@ clustersInfo:
   clusterInitialFailoverVersion:
     active: 0
     standby: 1
+
+kafkaCommon:
```

Comment:
Just curious is there any topics using it other than visibility-topic?

Reply:
Can we combine this config with kafka ? https://github.com/uber/cadence/blob/master/config/development_active.yaml#L77

It will be confusing to see two configs: kafka and kafkaCommon

Reply:
1. Yes, there will be at least topic for workflow insight. 
2. That could be done, but the difference is Kafka config for replication is cadence cluster specific, but for visibility is not. Also, there might be case people want to enable visibility to kafka but not enable cross dc. That's why I currently make it like this.

Reply:
Enabling cross DC is another config: https://github.com/uber/cadence/blob/master/config/development_active.yaml#L69
So it is fine to add other kafka config there